rust/src/writer/stats.rs

```diff
@@ -13,6 +14,7 @@ use parquet::{
 use super::*;
 use crate::action::{Add, ColumnValueStat, Stats};
 
+/// Creates an [`Add`] log action struct.
 pub(crate) fn create_add(
```

Comment:
this is still private to the crate, right? Was it more about the stats module?

Reply:
Sorry, that was a mistake. I am interested in making create_add public.